Sources/HaCWebsiteLib/ViewModels/GAScript.swift

```diff
@@ -0,0 +1,22 @@
+import HaCTML
+import Foundation
+import DotEnv
+
+struct GAScript: Nodeable {
+  let googleAnalyticsId = DotEnv.get("GOOGLE_ANALYTICS_ID")!
```

Comment:
I love love love that we're putting this in an environment variable. Where this would be the most useful, I think, is in development environments where we do not want analytics to be recorded. However, this will crash when the variable is not present.

Would it be more useful to instead not add the script at all if there is no GA ID set? This way we can easily enable it for production and developers don't have to do anything.

Reply:
This is a great point, you're absolutely right. I've updated the implementation to include this behaviour.